db/migrate/20240917142926_remove_foreign_keys_from_participant_id_changes.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveForeignKeysFromParticipantIdChanges < ActiveRecord::Migration[7.1]
+  def up
+    remove_foreign_key :participant_id_changes, column: :from_participant_id
```

Comment:
Is it worth renaming these to include `ecf_id` like we do for the other ECF ids?

Reply:
good q, I think worth discussing with the team, right now we're adding `ecf_id`s everywhere, after separation some of those will be generated automatically on creation as they are required by the API, some of them not. Eventually we might remove the unnecessary ones after separation from the db and rename the remaining `ecf_id`s to something else. For this one we could follow the same pattern and rename to include `ecf_id` for now 🤔

Reply:
`from_user_ecf_id` and `to_user_ecf_id`?

Reply:
maybe `from_participant_ecf_id` and `to_participant_ecf_id` instead?

Reply:
discussing in person with @leandroalemao: I think we should probably stick to the same name as they match what we have in the API for now, and we will probably rename the `ecf_id` after separation so doesn't feel like we need to change them here